api-catalog-services/src/main/java/org/zowe/apiml/apicatalog/security/SecurityConfiguration.java

```diff
@@ -173,23 +171,6 @@ public SecurityFilterChain basicAuthOrTokenAllEndpointsFilterChain(HttpSecurity
             }
             return http.build();
         }
-
```

Comment:
Why did this particular code disappear?

Reply:
I moved it to a separate class so all Standalone mode code is in one package.